src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -114,5 +114,10 @@
     "op": "replace",
     "path": "/ResourceTypes/AWS::Route53::RecordSet/Properties/TTL/PrimitiveType",
     "value": "Long"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::ResourceGroups::Group/Properties/Tags/ItemType",
+    "value": "Tag"
```

Comment:
leaving a comment just to call out this part is hand-written / not generated by the scripts